test/public/envs/overview.test.js

```diff
@@ -81,22 +81,26 @@ module.exports = () => {
 
     it('Should display the correct items counter at the bottom of the page', async () => {
         await goToPage(page, 'env-overview');
-        await waitForTimeout(100);
 
+        await page.waitForSelector('#firstRowIndex');
```

Comment:
Not needed anymore, expectInnerText wait for it by itself

Reply:
Done